README.md

```diff
@@ -62,6 +65,8 @@ The `err` parameter is the result of the `ejs` rendering step.
 
 Setting `customCss` to `true` will include the `pub/css/screen_overrides.css`,`pub/css/print_overrides.css` and `pub/css/theme_override.css` files, in which you can override any of the default Slate theme, to save you from having to alter the main css files directly. This should make syncing up with future Shins / Slate releases easier.
 
+Setting `inline` to `true` will inline all page resources (expect resources referenced via CSS, such as fonts) to output html. This way HTML can be used stand-alone, without needing any other resources. It will also set `minify` to `true`.
```

Comment:
'expect' should be 'except'.

Reply:
writing docs and tests at same time is not that easy it seems :/

Reply:
:smile: